### PR TITLE
Feature/documentation update

### LINKED ISSUE #11: Documentation is not updated with new features
A couple of new features were noted in 0.66 release notes but they were not documented in help (like 'move' checkbox state and how to ender a horizontal line).

I think it could be a good idea to include new feature tips for every version in help (or link to changed pages of documentation)

### PR DESCRIPTION
**Pull request by nasedil**
Closes #11.

### CODE REVIEW

File: data/manual/Help/Check_Boxes.txt [hunk 1]
```diff
@@ -11,8 +10,9 @@ Zim supports lists with checkboxes instead of bullets. These look like:
 	[*] item 2a
 	[x] item 2b
 [*] item3
+[>] item4
 
-As you can see in this example checkboxes can have 3 states: [ ] open, [*] checked as 'OK' and [x] checked as 'NOK'. States can be toggle by clicking the checkbox with either the left mouse button or using the keyboard with ''<F12>'' and ''<shift><F12>'' respectively.
+As you can see in this example checkboxes can have 4 states: [ ] open, [*] checked as completed, [x] checked as not completed, and [>] cheked as moved. States can be toggle by clicking the checkbox with either the left mouse button or using the keyboard with ''<F12>'' and ''<shift><F12>'' respectively.
```
Comment (kwill) [inline data/manual/Help/Check_Boxes.txt]: minor typo here: "cheked as moved"

File: HACKING/HACKING.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
 Content-Type: text/x-zim-wiki
 Wiki-Format: zim 0.4
-Creation-Date: 2014-07-31T20:56:17+02:00
```
Comment (jaap-karssenberg) [inline HACKING/HACKING.txt]: On merge will need to put back these creation dates - unintended change due to bug in zim 0.66 I suppose

Comment (nasedil) [inline HACKING/HACKING.txt]: @jaap-karssenberg should I do anything?

File: data/manual/Help/Check_Boxes.txt (hunk 1 above)
Comment (nasedil) [inline data/manual/Help/Check_Boxes.txt]: @kwill thanks, fixed it.

**Comment by jaap-karssenberg** [comment]
Became c1bb37297f654af2ef126ee8c7dbd45d6dface67

**Pull request closed without merging**